azurelinuxagent/ga/collect_logs.py

```diff
@@ -170,29 +175,33 @@ def _collect_logs(self):
         # Invoke the command line tool in the agent to collect logs, with resource limits on CPU and memory (RAM).
 
         systemd_cmd = [
-            "systemd-run", "--unit={0}".format(logcollector.CGROUPS_UNIT),
+            "systemd-run", "--property=CPUAccounting=no", "--property=MemoryAccounting=no",
```

Comment:
I'd add a comment explaining why this is set to "no"

Reply:
added comment.